### ISSUE TITLE
Error on https://flasharch.com/en/archive/play/399ef7feb6f1a06da87312e9e28c32a5

### ISSUE THREAD

**Issue by Razer-X172** [ruffle-rs/ruffle#7865]
# Error Info
Error name: Error
Error message: panicked at 'called `Option::unwrap()` on a `None` value', core/src/avm2/array.rs:61:37
Error stack:
```
Error: panicked at 'called `Option::unwrap()` on a `None` value', core/src/avm2/array.rs:61:37
    at W.n.wbg.__wbg_new_df6e6ab7a65c4c4d (https://flasharch.com/js/ruffle/2022-09-04/core.ruffle.d139e1b3878dae5e0858.js:1:5258)
    at std::sync::once::Once::call_once::{{closure}}::hd2dd5af86508d22e (wasm://wasm/016ca582:wasm-function[937]:0x2665c8)
    at std::sync::once::Once::call_inner::h00971925aa1adad3 (wasm://wasm/016ca582:wasm-function[1378]:0x2ef11d)
    at core::ops::function::FnOnce::call_once{{vtable.shim}}::hc24950feb424fb14 (wasm://wasm/016ca582:wasm-function[4809]:0x442bff)
    at std::panicking::rust_panic_with_hook::h84feca33bd4bd229 (wasm://wasm/016ca582:wasm-function[3563]:0x40942c)
    at core::panicking::panic_fmt::hc171d095bc4a492d (wasm://wasm/016ca582:wasm-function[4983]:0x448fb4)
    at core::panicking::panic::h3b7a8a1eaf47fc0f (wasm://wasm/016ca582:wasm-function[4893]:0x44434d)
    at <ruffle_core::avm2::object::array_object::ArrayObject as ruffle_core::avm2::object::TObject>::set_property_local::h65a944e5a0946fb1 (wasm://wasm/016ca582:wasm-function[1278]:0x2d5338)
    at <ruffle_core::avm2::object::Object as ruffle_core::avm2::object::TObject>::set_property::h117f22791359cd5e (wasm://wasm/016ca582:wasm-function[400]:0x123e18)
    at ruffle_core::avm2::activation::Activation::run_actions::h19c265dc142ce180 (wasm://wasm/016ca582:wasm-function[290]:0x25a9d)
```
AVM2 stack:
```
    at scaryama.transitions::FlashingBtn/onChange()
    at flash.events::EventDispatcher/dispatchEvent()
    at fl.transitions::Tween/setPosition()
    at fl.transitions::Tween/update()
    at fl.transitions::Tween/set time()
    at fl.transitions::Tween/nextFrame()
    at fl.transitions::Tween/onEnterFrame()
```

# Player Info
Allows script access: false

# Page Info
Page URL: https://flasharch.com/en/archive/play/399ef7feb6f1a06da87312e9e28c32a5
SWF URL: https://flasharch.com/cdn/uploads/archived/2020/10/17/e0a3edb5d66f12e00e161c4e254b0c26_20201017160245.swf

# Browser Info
User Agent: Mozilla/5.0 (X11; CrOS x86_64 14909.124.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/104.0.0.0 Safari/537.36
Platform: Linux x86_64
Has touch support: false

# Ruffle Info
Version: 0.1.0
Name: nightly 2022-09-04
Channel: nightly
Built: 2022-09-04T00:16:27.336Z
Commit: 3b752cc600297c101ce6abd768614c258e28317c
Is extension: false

# Metadata
width: 640
height: 480
frameRate: 45
numFrames: 1734
swfVersion: 20
backgroundColor: #FFFFFF
isActionScript3: true
uncompressedLength: 867283

**Comment by EmperorBale** [comment]
Fixed by #7857

**Comment by n0samu** [comment]
The newer report #8609 has the AVM2 stack info, in case that's helpful

**Comment by n0samu** [comment]
Fixed by #8623
